build-aux/flatpak/com.github.birros.WebArchives.yml

```
@@ -95,15 +96,8 @@ modules:
       - /share
     sources:
       - type: archive
-        url: https://snapshot.debian.org/archive/debian/20220920T060228Z/pool/main/libg/libgee-0.8/libgee-0.8_0.20.6.orig.tar.xz
+        url: https://download.gnome.org/sources/libgee/0.20/libgee-0.20.6.tar.xz
         sha256: 1bf834f5e10d60cc6124d74ed3c1dd38da646787fbf7872220b8b4068e476d4d
```

@TheEvilSkeleton Why did you choose version `0.20.6` of libgee instead of the more recent `0.20.8`? The version `0.20.8` seems to work.

```suggestion
        url: https://download.gnome.org/sources/libgee/0.20/libgee-0.20.8.tar.xz
        sha256: 189815ac143d89867193b0c52b7dc31f3aa108a15f04d6b5dca2b6adfad0b0ee
```

---

No reason in particular. I updated the manifest a week ago and haven't checked for updates since then.